edited gitignore to ignore more files

DustinKLo opened the pull request:
added function to calculate md5 checksum
api request to get md5 checksum from esa sci-hub
wrote md5 checksum to s1-iw_slc-....zip.md5
aded md5 checksum to es metadata
compared local md5 checksum to sci-hub's checkum and raised error if they do not match

pymonger commented:
Tested:

```
(verdi)[ops@ip-100-64-134-109 test_los_bug]$ docker run --rm -ti -v /data2:/data2 -w /data2/test/test_los_bug ffe /usr/local/gdal/bin/gdalinfo -stats los.
rdr.geo.vrt
Driver: VRT/Virtual Raster
Files: los.rdr.geo.vrt
       ./los.rdr.geo
Size is 11914, 7101
Coordinate System is:
GEOGCS["WGS 84",
    DATUM["WGS_1984",
        SPHEROID["WGS 84",6378137,298.257223563,
            AUTHORITY["EPSG","7030"]],
        AUTHORITY["EPSG","6326"]],
    PRIMEM["Greenwich",0,
        AUTHORITY["EPSG","8901"]],
    UNIT["degree",0.0174532925199433,
        AUTHORITY["EPSG","9122"]],
    AUTHORITY["EPSG","4326"]]
Origin = (-122.876666666666665,38.121666666666670)
Pixel Size = (0.000277777777778,-0.000277777777778)
Corner Coordinates:
Upper Left  (-122.8766667,  38.1216667) (122d52'36.00"W, 38d 7'18.00"N)
Lower Left  (-122.8766667,  36.1491667) (122d52'36.00"W, 36d 8'57.00"N)
Upper Right (-119.5672222,  38.1216667) (119d34' 2.00"W, 38d 7'18.00"N)
Lower Right (-119.5672222,  36.1491667) (119d34' 2.00"W, 36d 8'57.00"N)
Center      (-121.2219444,  37.1354167) (121d13'19.00"W, 37d 8' 7.50"N)
Band 1 Block=11914x1 Type=Float32, ColorInterp=Undefined
  Minimum=0.000, Maximum=45.895, Mean=17.586, StdDev=18.750
  Metadata:
    STATISTICS_MAXIMUM=45.8952293396
    STATISTICS_MEAN=17.585654965377
    STATISTICS_MINIMUM=0
    STATISTICS_STDDEV=18.749613093365
Band 2 Block=11914x1 Type=Float32, ColorInterp=Undefined
  Minimum=-101.001, Maximum=0.000, Mean=-47.415, StdDev=50.077
  Metadata:
    STATISTICS_MAXIMUM=0
    STATISTICS_MEAN=-47.415118891683
    STATISTICS_MINIMUM=-101.00118255615
    STATISTICS_STDDEV=50.077159787931
```

DustinKLo committed:
edited gitignore to ignore more files
added function to calculate md5 checksum
api request to get md5 checksum from esa sci-hub
wrote md5 checksum to s1-iw_slc-....zip.md5
aded md5 checksum to es metadata
compared local md5 checksum to sci-hub's checkum and raised error if they do not match
